src/Integration/WordPressEmails.php

```diff
@@ -145,12 +145,26 @@ public function disable_send_confirmation_on_admin_email() {
 
 	}
 
+	/**
+	 * Disables email on admin email address changed
+	 *
+	 * @action notification/init
+	 *
+	 * @since 8.0.15
```

Comment:
```suggestion
	 * @since [Next]
```

Do it this way, the `[Next]` placeholder will be replaced with the version number upon release